### DISCUSSION TITLE
[PROJECT] Debug / Dump server for Roadrunner

### DISCUSSION THREAD

**Discussion by khepin** [roadrunner-server/roadrunner#510] [Show and tell]
The debugging experience I've had in early stages of using roadrunner wasn't as good as I expected. I'm very much a fan of `dump` ing values where it makes sense to me to get my debug info and the info only being available mixed in with roadrunner's access log in a terminal wasn't great.

So I've been working on https://github.com/khepin/rr-dumpserver which allows retrieving and serving the output of `symfony/var-dumper` from a roadrunner ran program into a web UI served by the roadrunner go server.

Just wanted to let you know in case you find that interesting / valuable. Cheers.

**Comment by rustatian** [comment]
Awesome work @khepin. If you need any help from our side, please, let us know :)
